cadence/scripts/nftStorefront/get_listing_item.cdc

```diff
@@ -5,20 +5,22 @@ import KittyItems from "../../contracts/KittyItems.cdc"
 
 pub struct ListingItem {
     pub let name: String
-    pub let description: String
+    pub let image: String
     pub let thumbnail: String
+    pub let description: String
 
     pub let itemID: UInt64
     pub let resourceID: UInt64
-    pub let kind: KittyItems.Kind
-    pub let rarity: KittyItems.Rarity
+    pub let kind: UInt8
+    pub let rarity: UInt8
```

Comment:
Why did we revert from enums to ints?

Reply:
I'm not sure about this one.

Reply:
@psiemens I converted the KittyItem structs in the scripts to use UInt8. Their values were coming back as `{rawValue: 3}` so I thought it would be better to just return the int.

I'll change them back to structs

Reply:
I opened a PR here: https://github.com/onflow/kitty-items/pull/233